README.md

```diff
@@ -613,9 +614,11 @@ company’s profit to shareholders, authorized by the board of directors.
 Established public companies and some private companies pay dividends, but this is rare
 among startups and companies focused on rapid growth, since they often wish to re-invest
 their profits into expanding the business, rather than paying that money back to
-shareholders. For example, Amazon
-[has never](https://www.fool.com/investing/2017/12/28/will-amazon-start-paying-a-dividend-in-2018.aspx)
-paid dividends.
+shareholders.
+
+- Amazon, for example,
```

Comment:
Same here. Either top-level elaborative paragraph, or (as discussed) also think it's OK in this case just to keep it part of the definition since it's such an important example.